h5py/tests/old/test_h5pl.py

```diff
@@ -0,0 +1,56 @@
+# This file is part of h5py, a Python interface to the HDF5 library.
+#
+# http://www.h5py.org
+#
+# Copyright 2008-2019 Andrew Collette and contributors
+#
+# License:  Standard 3-clause BSD; see "license.txt" for full license terms
+#           and contributor agreement.
+
+from __future__ import absolute_import
+
+try:
+    import unittest2 as ut
+except ImportError:
+    import unittest as ut
+
+import h5py
+from h5py import h5pl
+
+from ..common import TestCase
+
+@ut.skip('The tests have side effects')
```

Comment:
If we go for this, I think we should come up with a way to run the tests by default; maybe run them in a separate process? Tests that don't run by default aren't very helpful.

Reply:
Tests are also documentation ;-)
But I agree; see my other comment.